Godeps/Godeps.json

```diff
@@ -637,6 +637,438 @@
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/authorization/v1beta1",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/autoscaling",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/autoscaling/v1",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/autoscaling/v2alpha1",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/batch",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/batch/v1",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/batch/v2alpha1",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/certificates",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/certificates/v1beta1",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/extensions",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/extensions/v1beta1",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/policy",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/policy/v1beta1",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/rbac",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/rbac/v1alpha1",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/rbac/v1beta1",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/settings",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/settings/v1alpha1",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/storage",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/storage/v1",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/apis/storage/v1beta1",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/util",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/util/parsers",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/version",
```

Comment:
Or is this Godeps.json only, i.e. without the actual vendor/ files? Then it's ok.